plugins/mcreator-themes/themes/default_dark/theme.json

```diff
@@ -1,3 +1,11 @@
 {
-  "name": "Default Dark"
+  "name": "Default Dark",
+  "colorScheme": {
+    "id": "light",

```

Comment:
Dark theme should have dark id I thik, but I don't think this ID is even needed. Consider removing this field.